pf9/support/generate_bundle.py

```diff
@@ -108,8 +114,8 @@ def upload_pf9cli_logs (self, du_url, host):
         du_url = du_url.replace("https://","")
         header = 'x-amz-acl:bucket-owner-full-control'
         S3_location = "https://s3-us-west-2.amazonaws.com/loguploads.platform9.com/"+str(du_url)+"/"+str(host)+"/"
-        cmd = subprocess.run(["curl", "-T", filename, "-H", header, S3_location])
-        cmd = subprocess.run(["curl", "-T", cli_logs, "-H", header, S3_location])
+        cmd = subprocess.call(["curl", "-T", filename, "-H", header, S3_location], stdout=DEVNULL, stderr=DEVNULL)
```

Comment:
AFAIK This just gets logged and does not show the error to user. Redirecting to /dev/null will suppress that info.

Reply:
Yes. That's the intent. I want to suppress any messages related to support log files. Treat it more as best effort attempt.